Kinder/Classes/FileManager.cs

```diff
@@ -13,45 +13,22 @@ class FileManager
         static string path = System.IO.Directory.GetParent(System.IO.Directory.GetParent(System.IO.Directory.GetParent(Environment.CurrentDirectory).ToString()).ToString()).ToString();
         static string fileLocation = System.IO.Path.Combine(path, "Data_files\\Users.txt");
         static string fileLocation_liked = System.IO.Path.Combine(path, "Data_files\\Items_liked.txt");
-        public static List<User> GetUsers()
+
+        //Dependency injection B
+        public IGetItems parsingOperationItems;
+
+        public List<Item> GetAllItems(IGetItems _parsingOperations)
```

Comment:
It will work exactly in the same way, but we will have some additional consistency: 
parse operations -> file manager(to do those operations) -> do things with data that we got

Reply:
Oh, okay. Understood :)